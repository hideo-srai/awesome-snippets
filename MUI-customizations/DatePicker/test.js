// @flow
import { start } from '../../testing'

describe('DateTimePicker', () => {
  it('renders examples', () => start().examples())
})
