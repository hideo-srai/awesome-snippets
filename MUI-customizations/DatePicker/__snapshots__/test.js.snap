// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateTimePicker renders examples: lg 1`] = `
Array [
  <span
    className="tdd-Text--body"
  >
    November 18, 2018
  </span>,
  " ",
  <button
    className="tdd-MuiButtonBase-root tdd-MuiButton-root tdd-MuiButton-contained tdd-MuiButton-containedPrimary tdd-MuiButton-raised tdd-MuiButton-raisedPrimary tdd-Button--button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="tdd-MuiButton-label"
    >
      Pick Date
    </span>
    <span
      className="tdd-MuiTouchRipple-root"
    />
  </button>,
  <button
    className="tdd-MuiButtonBase-root tdd-MuiButton-root tdd-MuiButton-contained tdd-MuiButton-containedPrimary tdd-MuiButton-raised tdd-MuiButton-raisedPrimary tdd-Button--button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="tdd-MuiButton-label"
    >
      Pick Date - November 18, 2018
    </span>
    <span
      className="tdd-MuiTouchRipple-root"
    />
  </button>,
]
`;

exports[`DateTimePicker renders examples: md 1`] = `
Array [
  <span
    className="tdd-Text--body"
  >
    November 18, 2018
  </span>,
  " ",
  <button
    className="tdd-MuiButtonBase-root tdd-MuiButton-root tdd-MuiButton-contained tdd-MuiButton-containedPrimary tdd-MuiButton-raised tdd-MuiButton-raisedPrimary tdd-Button--button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="tdd-MuiButton-label"
    >
      Pick Date
    </span>
    <span
      className="tdd-MuiTouchRipple-root"
    />
  </button>,
  <button
    className="tdd-MuiButtonBase-root tdd-MuiButton-root tdd-MuiButton-contained tdd-MuiButton-containedPrimary tdd-MuiButton-raised tdd-MuiButton-raisedPrimary tdd-Button--button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="tdd-MuiButton-label"
    >
      Pick Date - November 18, 2018
    </span>
    <span
      className="tdd-MuiTouchRipple-root"
    />
  </button>,
]
`;

exports[`DateTimePicker renders examples: sm 1`] = `
Array [
  <span
    className="tdd-Text--body"
  >
    November 18, 2018
  </span>,
  " ",
  <button
    className="tdd-MuiButtonBase-root tdd-MuiButton-root tdd-MuiButton-contained tdd-MuiButton-containedPrimary tdd-MuiButton-raised tdd-MuiButton-raisedPrimary tdd-Button--button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="tdd-MuiButton-label"
    >
      Pick Date
    </span>
    <span
      className="tdd-MuiTouchRipple-root"
    />
  </button>,
  <button
    className="tdd-MuiButtonBase-root tdd-MuiButton-root tdd-MuiButton-contained tdd-MuiButton-containedPrimary tdd-MuiButton-raised tdd-MuiButton-raisedPrimary tdd-Button--button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="tdd-MuiButton-label"
    >
      Pick Date - November 18, 2018
    </span>
    <span
      className="tdd-MuiTouchRipple-root"
    />
  </button>,
]
`;

exports[`DateTimePicker renders examples: xl 1`] = `
Array [
  <span
    className="tdd-Text--body"
  >
    November 18, 2018
  </span>,
  " ",
  <button
    className="tdd-MuiButtonBase-root tdd-MuiButton-root tdd-MuiButton-contained tdd-MuiButton-containedPrimary tdd-MuiButton-raised tdd-MuiButton-raisedPrimary tdd-Button--button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="tdd-MuiButton-label"
    >
      Pick Date
    </span>
    <span
      className="tdd-MuiTouchRipple-root"
    />
  </button>,
  <button
    className="tdd-MuiButtonBase-root tdd-MuiButton-root tdd-MuiButton-contained tdd-MuiButton-containedPrimary tdd-MuiButton-raised tdd-MuiButton-raisedPrimary tdd-Button--button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="tdd-MuiButton-label"
    >
      Pick Date - November 18, 2018
    </span>
    <span
      className="tdd-MuiTouchRipple-root"
    />
  </button>,
]
`;

exports[`DateTimePicker renders examples: xs 1`] = `
Array [
  <span
    className="tdd-Text--body"
  >
    November 18, 2018
  </span>,
  " ",
  <button
    className="tdd-MuiButtonBase-root tdd-MuiButton-root tdd-MuiButton-contained tdd-MuiButton-containedPrimary tdd-MuiButton-raised tdd-MuiButton-raisedPrimary tdd-Button--button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="tdd-MuiButton-label"
    >
      Pick Date
    </span>
    <span
      className="tdd-MuiTouchRipple-root"
    />
  </button>,
  <button
    className="tdd-MuiButtonBase-root tdd-MuiButton-root tdd-MuiButton-contained tdd-MuiButton-containedPrimary tdd-MuiButton-raised tdd-MuiButton-raisedPrimary tdd-Button--button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="tdd-MuiButton-label"
    >
      Pick Date - November 18, 2018
    </span>
    <span
      className="tdd-MuiTouchRipple-root"
    />
  </button>,
]
`;
